requirements/base.txt

```diff
@@ -2,6 +2,10 @@
 # After the first "pip install -r", just run "pip freeze" and add the version
 # to each package in each requirements/*.txt.
 
+# Deps of current cffi (must be missing from ustream reqs)
+appdirs
+pyparsing
```

Comment:
we're pinning dependencies everywhere else so that we don't get major changes slipping in without being aware